Tests/Tests/STPApplePayTest.m

```diff
@@ -49,6 +49,9 @@ - (void)testCanSubmitPaymentRequestReturnsNOIfMerchantIdentifierIsNil {
 }
 
 - (void)testJCBPaymentNetwork {
+    if (&PKPaymentNetworkJCB == NULL) {
+        return;
```

Comment:
If you wanted to be *really* complete you could instead test/assert that `setJCBPaymentNetworkSupported:YES` doesn't crash/add garbage/NULL to the `supportedPKPaymentNetworks` response.

Probably not worth it.

Reply:
Good idea!  `[Stripe supportedPKPaymentNetworks]` crashed in an earlier commit because I forgot this check.